addons/arsenalExport/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\potato\addons\arsenalExport
```

Comment:
Could go either place.
There are no code dependencies so I like having it separate which leaves each module cleaner.